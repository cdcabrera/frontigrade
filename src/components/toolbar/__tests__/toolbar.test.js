import React from 'react';
import { mount } from 'enzyme';
import Toolbar from '../toolbar';

describe('Toolbar Component', () => {
  it('should render', () => {
    const props = {};

    const component = mount(<Toolbar {...props} />);

    expect(component.render()).toMatchSnapshot();
  });
});
