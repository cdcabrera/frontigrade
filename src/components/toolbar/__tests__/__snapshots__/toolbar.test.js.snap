// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toolbar Component should render 1`] = `
<div
  class="container-fluid"
>
  <div
    class="toolbar-pf row"
  >
    <div
      class="col-sm-12"
    >
      <form
        class="toolbar-pf-actions"
      >
        <div
          class="toolbar-pf-action-right"
        />
      </form>
      <div
        class="toolbar-pf-results row"
      >
        <div
          class="col-sm-12"
        >
          <h5
            class="cloudmeter-view-count"
          >
            Â 
          </h5>
        </div>
      </div>
    </div>
  </div>
</div>
`;
